### ISSUE TITLE
suggestion: add tvos support

### ISSUE THREAD

**Issue by lemonhead94** [SuperMarcus/NineAnimator#44]
I saw that you were eager to dive into catalyst...
Since you did such a nice MVC separation maybe in the future you are willing to add another UI for tvOS :)

**Comment by SuperMarcus** [comment]
You're right! I would like to see this app running on tvOS as well, and the model layer of the app is indeed designed to run on different SDKs (even on macOS natively with AppKit).

However, since I can't get my hands on an Apple TV 4+ right now, this is certainly not something that's coming anytime soon (it's another story if anyone wants to collaborate with me on this).

**Comment by Catta1997** [comment]
Do you need somebody that can test it or somebody that know how to “write” it? Because I have an Apple TV 4K but i don’t know how to write an app for apple tv

**Comment by SuperMarcus** [comment]
@Catta1997 Have others to write the code and create PRs may be a better idea.

I am still willing to write the code for tvOS and test run it in a simulator. And, of course, I'll let you know if this happens. I'm only worried because this process will be tedious and error-prone.

**Comment by jennataiga** [comment]
@SuperMarcus were you be able to get a apple tv 4. ? It would be really awesome if you somehow release a tvOS version
